### ISSUE TITLE
Adding SCIP package in simple words

### ISSUE THREAD

**Issue by Reza-rn** [SCIP-Interfaces/SCIP.jl#57]
Hi, 
Could anybody describe me how I can add "SCIP" solver to the Julia in very simple words. I have just started working with Julia and README file description seems too complicated to me.
Thanks.

**Comment by leethargo** [comment]
Unfortunately, the installation is not completely automatic and the steps described in the README are necessary. To help you better, could you answer these questions:

- what operating system do you use?
- have you managed to download and compile the SCIP solver itself?
- what steps from the README have you tried, or do need explanation of?

**Comment by Reza-rn** [comment]
Thanks for answering.
- My operating system is windows 10
- I am using both Julia 0.5.1 and 0.6.0
- I need to solve my optimization problem in Julia since I implemented it in Julia. So, I have not tried to use compiler itself
- The installation consists of 4 main steps:

1. there is two codes in README to download and untar SCIP as below: 
> wget http://scip.zib.de/download/release/scipoptsuite-4.0.0.tgz
> tar xzf scipoptsuite-4.0.0.tgz
non of these codes works for me in Julia. In fact, I downloaded the SCIP from its website manually and unzip it. Then, what should I do next?

2. I do not know how to make share library. How I can use the below code:
> make SHARED=true GMP=false READLINE=false ZLIB=false scipoptlib

3. Accessing library: what should I need exactly in this step.

4. step 4 is for adding package in Julia which might be accomplished by using below code:
Pkg.add("SCIP")
